_dev/assets/js/jquery-wrap.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
```

Comment:
Shouldn't this be written as `(function($) {`?
And, the end of the scope as `})(jQuery)`?